base/test/src/main/java/org/openscience/cdk/test/interfaces/AbstractMappingTest.java

```diff
@@ -22,6 +23,7 @@ public abstract class AbstractMappingTest extends AbstractChemObjectTest {
     /**
      * Method to test whether the class complies with RFC #9.
      */
+    @Test
```

Comment:
Ah, I need to check, but this annotation is wrong. These tests should not be run separately, but are inherited by subclasses. I'm adding while going through commit one by one, so the final patch may not actually have the added @Test here.

Reply:
@johnmay, are you sure that this class is not now run as a separate class too?

Reply:
It think it was abstract, check the log files with the verify/sed file